samples/hello-oboe/src/main/java/com/google/sample/oboe/hellooboe/MainActivity.java

```diff
@@ -81,18 +83,34 @@ public boolean onTouchEvent(MotionEvent event) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
+        mLatencyText = findViewById(R.id.latencyText);
         setupAudioApiSpinner();
         setupPlaybackDeviceSpinner();
         setupChannelCountSpinner();
         setupBufferSizeSpinner();
 
-        // initialize native audio system
+    }
+    /*
+    * Creating engine in onResume() and destroying in onPause() so the stream retains exclusive
+    * mode only while in focus. This allows other apps to reclaim exclusive stream mode.
+    */
+    @Override
+    protected void onResume() {
+        super.onResume();
         PlaybackEngine.create(this);
-
-        // Periodically update the UI with the output stream latency
-        mLatencyText = findViewById(R.id.latencyText);
         setupLatencyUpdater();
+        // Return to defaults
+        mChannelCountSpinner.setSelection(1);
+        mPlaybackDeviceSpinner.setSelection(0);
+        mBufferSizeSpinner.setSelection(0);
+        mAudioApiSpinner.setSelection(0);
+    }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
```

Comment:
It is generally better to tear down in reverse order or startup. So move call to super.onPause() to end of onPause().